modules/artifact/src/test/java/org/mule/runtime/module/artifact/classloader/IBMMQResourceReleaserTestCase.java

```diff
@@ -333,4 +518,51 @@ private int countJULKnownLevels(ClassLoader artifactClassLoader) throws Exceptio
     return counter;
   }
 
+  private int countWorkerThreads(ClassLoader artifactClassLoader) {
+    ThreadUtils.getAllThreads().stream().forEach(o -> {
+      System.out.println(o.getName());
+    });
```

Comment:
Remove. It is not a good idea (although not always avoidable), to print stuff in a successful test run.